portalnet/src/overlay_service.rs

```diff
@@ -1125,7 +1124,8 @@ where
 
             let mut data = vec![];
             if let Err(err) = stream.read_to_eof(&mut data).await {
-                warn!(%err, cid.send, cid.recv, peer = ?cid.peer.client(), "error reading data from uTP stream");
+                error!(%err, cid.send, cid.recv, peer = ?cid.peer.client(), "error reading data from uTP stream");
```

Comment:
I think this log level will eventually drop down to at most a warning, and maybe even lower. In general, if trin is behaving correctly and a peer is doing something wrong, I don't think that belongs as an `ERROR` in our logs. 

But I'm fine with slightly over-aggressive logs early in the project lifecycle.